includes/class-indieauth-client-discovery.php

```diff
@@ -5,6 +5,7 @@ class IndieAuth_Client_Discovery {
 	protected $manifest = array();
 	protected $html     = array();
 	protected $mf2      = array();
+	protected $json     = array();
```

Comment:
You define this but then don't use it. Required?

Reply:
Fixed that. It is meant to hold the json for returning.